pre-index.php

```diff
@@ -147,6 +147,12 @@ function pokemonFilterImages($noPokemonNumbers)
                  title="PayPal - The safer, easier way to pay online!" alt="Donate" style="float: right;">
         </a>';
         }
+        if ($helplink != "") {
+            echo '<a href="' . $helplink . '" target="_blank" style="margin-bottom: 5px; vertical-align: middle; padding:0 5px;">
```

Comment:
Move CSS style into one of the SCSS files.

Reply:
I copied and pasted this from the paypalurl/discordurl above it, do you want me to move those into stylesheets as well so they can be consistent?